graph/src/data/subgraph/schema.rs

```diff
@@ -112,10 +113,12 @@ impl DeploymentCreate {
     pub fn new(
         source_manifest: &SubgraphManifest<impl Blockchain>,
         earliest_block: Option<BlockPtr>,
+        latest_block: Option<BlockPtr>,
     ) -> Self {
         Self {
             manifest: SubgraphManifestEntity::from(source_manifest),
             earliest_block: earliest_block.cheap_clone(),
+            latest_block: latest_block.cheap_clone(),
```

Comment:
When creating _new_ deployments, we previously passed the start block in via `latest_block` and there was no `earliest_block`. This could potentially be simplified by just removing the `earliest_block` parameter and field entirely.